<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система рейтингування</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TabsWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="IniDataTab">
       <attribute name="title">
        <string>Додавання студентів</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>531</width>
          <height>341</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="4" rowspan="2">
          <widget class="QPushButton" name="ResetButton">
           <property name="text">
            <string>Скинути список</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="CoefLabel">
           <property name="text">
            <string>Коефієнт</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="M1Edit"/>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="NameEdit"/>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="M3Edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="M2label">
           <property name="text">
            <string>Оцінка 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="M2Edit"/>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QLineEdit" name="M4Edit"/>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="RateButton">
           <property name="text">
            <string>Розрахувати рейтинг</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLineEdit" name="M5Edit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="NameLabel">
           <property name="text">
            <string>Ім'я</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="AddButton">
           <property name="text">
            <string>Додати студента</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="3">
          <widget class="QLabel" name="BudgetLabel">
           <property name="text">
            <string>Бюджетник?</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="CoefEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="M5label">
           <property name="text">
            <string>Оцінка 5</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="M4label">
           <property name="text">
            <string>Оцінка 4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="M3label">
           <property name="text">
            <string>Оцінка 3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="M1label">
           <property name="text">
            <string>Оцінка 1</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLineEdit" name="BudgetEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="InputTab">
       <attribute name="title">
        <string>Вхідний список</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="InputList">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AvgTab">
       <attribute name="title">
        <string>Середній бал</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="AvgList">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SortTab">
       <attribute name="title">
        <string>Відсортований список</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="SortList">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RatingTab">
       <attribute name="title">
        <string>Рейтинг</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="RateList">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
